scripts/scripts.js

```diff
@@ -308,6 +308,12 @@ export function decorateExternalAnchors(externalAnchors) {
  */
 export function decorateAnchors(element = document) {
   const anchors = element.getElementsByTagName('a');
+  Array.from(anchors).forEach((a) => {
+    // check if a has a atribute aria-label
+    if (!a.getAttribute('aria-label')) {
+      a.setAttribute('aria-label', a.innerText.trim() || 'Link');
```

Comment:
I don't think Word can add this attribute to a link. Additionally "Link" isn't a sufficient for Accessibility, so I don't think this change is really doing anything for us.